test/unit/worldwide_ab_test_helper_test.rb

```diff
@@ -0,0 +1,63 @@
+require 'test_helper'
+
+class WorldwideAbHelperTest < ActiveSupport::TestCase
+  def subject
+    WorldwideAbTestHelper.new
+  end
+
+  test "content returns the parsed file " do
+    result = subject.content
+    assert result[:sample].present?
```

Comment:
We're relying on the actual yml file here, so it sort of acts like a fixture. We could change the `WorldwideAbTestHelper` class so that we initialise it with some dummy content instead. Something like:

```
dummy_content = {
  sample: {
    something: "stuff"
  }
}

WorldwideAbTestHelper.new(dummy_content)
```

That might make the tests clearer in terms of what data is legitimate and what isn't. Although that would just be changing the code to aid testing, so not sure if there's any benefit to it (other than someone reading this having to look at the yml file). Probably not a big deal, the reliance on the yml file just jumped out at me...